code/obj/item/toys.dm

```diff
@@ -932,6 +932,8 @@ var/list/figure_patreon_rarity = list(\
 	stamina_damage = 1
 	stamina_cost = 1
 	stamina_crit_chance = 1
+	var/redeemer = null
+	var/receiver = null
```

Comment:
You probably want to do ckeys instead of mobs, so that way you won't have any lingering references (well, one less) in case the mobs get deleted.

Reply:
Added in latest commit